src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/courtsentencing/CourtCasesToNomisIntTest.kt

```diff
@@ -120,6 +122,12 @@ class CourtCasesToNomisIntTest : SqsIntegrationTestBase() {
                 WireMock.equalTo("A"),
               ),
             )
+            .withRequestBody(
+              WireMock.matchingJsonPath(
```

Comment:
very minor but you might want to do some imports for `WireMock` static for better readbilitty